apps/service_providers/migrations/0019_auto_20241029_0901.py

```diff
@@ -12,23 +12,64 @@ def _create_llm_provider_models(apps, schema_editor):
     _create_default_llm_provider_models(apps, schema_editor)
     _create_custom_llm_provider_models(apps, schema_editor)
 
-DefaultLlmProviderModel = namedtuple('DefaultLlmProviderModel', ["type", "name", "max_token_limit"])
 
 DEFAULT_LLM_PROVIDER_MODELS = {
-    DefaultLlmProviderModel("openai", "gpt-4o-mini", 8192),
-    DefaultLlmProviderModel("anthropic", "claude", 8192),
+    "azure": [
+        "gpt-4o-mini",
+        "gpt-4o",
+        "gpt-4",
+        "gpt-4-32k",
+        "gpt-35-turbo",
+        "gpt-35-turbo-16k",
+    ],
+    "anthropic": [
+        "claude-3-5-sonnet-latest",
+        "claude-3-5-sonnet-20241022",
+        "claude-3-5-sonnet-20240620",
+        "claude-3-opus-20240229",
+        "claude-3-sonnet-20240229",
+        "claude-3-haiku-20240307",
+        "claude-2.0",
+        "claude-2.1",
+        "claude-instant-1.2"
+    ],
+    "openai": [
+        "gpt-4o-mini",
+        "gpt-4o-mini-2024-07-18",
+        "gpt-4o",
+        "gpt-4o-2024-08-06",
+        "gpt-4o-2024-05-13",
+        "o1-preview",
+        "o1-preview-2024-09-12",
+        "o1-mini",
+        "o1-mini-2024-09-12",
+        "gpt-4",
+        "gpt-4-turbo",
+        "gpt-4-turbo-preview",
+        "gpt-4-0125-preview",
+        "gpt-4-1106-preview",
+        "gpt-4-0613",
+        "gpt-4-32k",
+        "gpt-4-32k-0613",
+        "gpt-3.5-turbo",
+        "gpt-3.5-turbo-0125",
+        "gpt-3.5-turbo-1106",
+        "gpt-3.5-turbo-0613",
+        "gpt-3.5-turbo-16k",
+        "gpt-3.5-turbo-16k-0613"
+    ]
```

Comment:
Nice, where did this list come from? Is there more metadata?

Reply:
It's a modified version of this: https://raw.githubusercontent.com/FlowiseAI/Flowise/main/packages/components/models.json

I only included the ones supported in OCS (anthropic, openai, azure openai). The only extra metadata is a `label` field, which is mostly the same as the `name`, except for things like `gpt-4o`, which say `"label": "gpt-4o (latest)"`. 

We could probably add a label field to the model if we wanted, but this is getting pretty large now, so we could do it separately.